.github/actions/install-global-turbo/action.yml

```diff
@@ -6,8 +6,7 @@ runs:
   steps:
     - name: Install Turbo globally
       shell: bash
-      # Until we are migrate ourselves to 2.0 we pin to the latest 1.x release
       run: |
-        VERSION=$(npm view turbo --json | jq -r '.versions | last')
+        VERSION=$(npm view turbo --json | jq -r '.versions | map(select(test("2."))) | last')
```

Comment:
Versions are ordered by release date. We need to only select 2.x releases in order to prevent grabbing a 1.x release that might occur later.